state/store.go

```diff
@@ -23,6 +23,10 @@ func calcABCIResponsesKey(height int64) []byte {
 	return []byte(fmt.Sprintf("abciResponsesKey:%v", height))
 }
 
+func calSoftwareHeight(height int64) []byte {
```

Comment:
calc

It seems a kind suggest changes? Did the repo this is coming from disapear?